code/modules/mob/transform_procs.dm

```diff
@@ -1,7 +1,6 @@
 /mob/living/carbon/human/proc/monkeyize()
 	var/mob/H = src
 	H.dna.SetSEState(MONKEYBLOCK,1)
-	domutcheck(H, null)

```

Comment:
To be clear, there's also changes to changeling procs that you shouldn't need now. The idea is to change as few existing things as possible, to minimize the odds of accidentally breaking something.